tests/wpt/mozilla/tests/mozilla/interfaces.html

```diff
@@ -141,27 +141,19 @@
   "ImageData",
   "Image",
   "KeyboardEvent",
-  "Location",
-  "MediaError",
   "MediaList",
   "MediaQueryList",
   "MediaQueryListEvent",
   "MessageEvent",
-  "MimeType",
-  "MimeTypeArray",
   "MouseEvent",
   "NamedNodeMap",
   "Navigator",
   "Node",
   "NodeFilter",
   "NodeIterator",
   "NodeList",
-  "PageTransitionEvent",
   "Performance",
   "PerformanceTiming",
-  "Plugin",
-  "PluginArray",
-  "PopStateEvent",
```

Comment:
-  "Plugin",
-  "PluginArray"
-  "PopStateEvent"

had also their exposition modified and don't have the `Exposed=Window` annotation any more. 
So I removed their occurrences from both test files.
What am I missing ?